docs-src/pages/UiTabs.vue

```diff
@@ -66,18 +64,22 @@
             <h4 class="page__demo-title">Custom header with HTML</h4>
 
             <ui-tabs fullwidth>
-                <ui-tab :key="tab.title" v-for="tab in demoTabs">
+                <ui-tab :key="tab.title" v-for="tab in demoTabs" :icon="tab.icon">
                     <template #header>
                         <div class="my-custom-tab-header">
                             <ui-icon :icon="tab.icon"></ui-icon>
                             <span>{{ tab.title }}: <i>HTML</i></span>
                         </div>
                     </template>
 
-                    <template #icon>
-                        <ui-icon :icon="tab.icon"></ui-icon>
-                    </template>
+                    <p>{{ tab.title }}</p>
+                </ui-tab>
+            </ui-tabs>
+
+            <h4 class="page__demo-title">Tabs with tooltips</h4>
```

Comment:
This example doesn't work for me. Perhaps it accidentally included from #539? Let's remove it from this PR.